Assets/Plugins/iOS/EOSManager_iOS.cs

```diff
@@ -0,0 +1,221 @@
+        {
+            Debug.Log(platformOptions.ProductId);
+            return Epic.OnlineServices.Platform.PlatformInterface.Create(ref (platformOptions as EOSiOSOptions).options);
+        }
+
+        //-------------------------------------------------------------------------
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <returns></returns>
+        public IEOSInitializeOptions CreateSystemInitOptions()
+        {
+            return new EOSiOSInitializeOptions();
+        }
+
+        //-------------------------------------------------------------------------
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="initializeOptions"></param>
+        /// <param name="configData"></param>
+        public void ConfigureSystemInitOptions(ref IEOSInitializeOptions initializeOptionsRef, EOSConfig configData)
+        {
+            Debug.Log("ConfigureSystemInitOptions");
+
+            EOSiOSInitializeOptions initializeOptions = (initializeOptionsRef as EOSiOSInitializeOptions);
+
+            if (initializeOptions == null)
+            {
+                throw new Exception("ConfigureSystemInitOptions: initializeOptions is null!");
+            }
+
+            if (GetEOS_iOSConfig() != null)
+            {
+                Debug.Log("GetEOS_iOSConfig() is not null");
+                if (initializeOptions.OverrideThreadAffinity.HasValue)
+                {
+                    var overrideThreadAffinity = initializeOptions.OverrideThreadAffinity.Value;
+                    overrideThreadAffinity.NetworkWork = iOSConfig.overrideValues.GetThreadAffinityNetworkWork(overrideThreadAffinity.NetworkWork);
+                    overrideThreadAffinity.StorageIo = iOSConfig.overrideValues.GetThreadAffinityStorageIO(overrideThreadAffinity.StorageIo);
+                    overrideThreadAffinity.WebSocketIo = iOSConfig.overrideValues.GetThreadAffinityWebSocketIO(overrideThreadAffinity.WebSocketIo);
+                    overrideThreadAffinity.P2PIo = iOSConfig.overrideValues.GetThreadAffinityP2PIO(overrideThreadAffinity.P2PIo);
+                    overrideThreadAffinity.HttpRequestIo = iOSConfig.overrideValues.GetThreadAffinityHTTPRequestIO(overrideThreadAffinity.HttpRequestIo);
+                    overrideThreadAffinity.RTCIo = iOSConfig.overrideValues.GetThreadAffinityRTCIO(overrideThreadAffinity.RTCIo);
+                    initializeOptions.OverrideThreadAffinity = overrideThreadAffinity;
+                }
+            }
+        }
+
+        //-------------------------------------------------------------------------
+        public IEOSCreateOptions CreateSystemPlatformOption()
+        {
+            var createOptions = new EOSiOSOptions();
+
+            return createOptions;
+        }
+
+        //-------------------------------------------------------------------------
+        /// <summary>
+        /// On iOS, this method handles looking up where the RTC options are.
```

Comment:
This comment appears to be copied and pasted from somewhere else and should be removed.

Reply:
Removed